welcome/oke_about.adoc

```diff
@@ -344,7 +344,6 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Red Hat Integration - Service Registry Operator | Not Included - Requires separate subscription | Not Included - Requires separate subscription | Service Registry
 | API Designer provided by Red Hat | Not Included - Requires separate subscription | Not Included - Requires separate subscription | API Designer
 | JBoss EAP provided by Red Hat | Not Included - Requires separate subscription | Not Included - Requires separate subscription | JBoss EAP
-| JBoss Web Server provided by Red Hat | Not Included - Requires separate subscription | Not Included - Requires separate subscription | JBoss Web Server
```

Comment:
This line should remain unchanged by this PR and the line 314 should be removed.
The final state of the JWS row should be like this:
`| JBoss Web Server provided by Red Hat | Not Included - Requires separate subscription | Not Included - Requires separate subscription | JBoss Web Server
`

Reply:
Hi @rolfedh , I've added a comment on how the entry for JWS should look like, thanks!

Reply:
Hello, after chat with zeke we found out that separate subscription is not needed for OCP users, so the right one is:
| JBoss Web Server provided by Red Hat | Not Included | Included | JWS Operator

Reply:
Thanks for all your comments. I've updated the document accordingly. 
@vassilismourikis please review and approve. Thank you.